### PR TITLE
testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- update to 25.0.4
- bump opcache.interned_strings_buffer to 32 #535 
- #542
- #549

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: doc/DESCRIPTION.md [hunk 1]
```diff
@@ -1 +1,12 @@
 Nextcloud Hub is a fully open-source on-premises content collaboration platform. Teams access, share and edit their documents, chat and participate in video calls and manage their mail and calendar and projects across mobile, desktop and web interfaces.
+
+### YunoHost features:
```
Comment (alexAubin) [inline doc/DESCRIPTION.md:3]:
```suggestion
### YunoHost-specific features
```

File: doc/DESCRIPTION_fr.md [hunk 2]
```diff
@@ -1 +1,11 @@
-Nextcloud Hub est la plate-forme de collaboration de contenu sur site entièrement open source. Les équipes accèdent, partagent et modifient leurs documents, discutent et participent à des appels vidéo et gèrent leur courrier, leur calendrier et leurs projets sur des interfaces mobiles, de bureau et Web.
\ No newline at end of file
+Nextcloud Hub est la plate-forme de collaboration de contenu sur site entièrement open source. Les équipes accèdent, partagent et modifient leurs documents, discutent et participent à des appels vidéo et gèrent leur courrier, leur calendrier et leurs projets sur des interfaces mobiles, de bureau et Web.
+
+### Caractéristiques spécifiques YunoHost :
```
Comment (alexAubin) [inline doc/DESCRIPTION_fr.md:3]:
```suggestion
### Caractéristiques spécifiques YunoHost
```

**Review by alexAubin — APPROVED** [review]

**Comment by alexAubin** [comment]
FYI I'm cancelling https://ci-apps-dev.yunohost.org/ci/job/5615 to move ci-apps-dev to another machine but the job was pretty much successful (running change-url which apparently worked and shouldnt be impacted by the code changes anyway)

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by lapineige** [comment]
Upgrade went fine :)

**Review by alexAubin — APPROVED** [review]
:+1:

**Pull request merged**